lib/ret_web/endpoint.ex

```diff
@@ -7,11 +7,35 @@ defmodule RetWeb.Endpoint do
   def get_cors_origins, do: Application.get_env(:ret, RetWeb.Endpoint)[:allowed_origins] |> String.split(",")
   def get_cors_origin_urls, do: get_cors_origins() |> Enum.filter(&(&1 != "*")) |> Enum.map(&URI.parse/1)
 
+  def local_address_ranges(), do: [CIDR.parse("192.168.0.0/16"), CIDR.parse("10.0.0.0/8"), CIDR.parse("172.16.0.0/12")]
```

Comment:
Can I make this a constant or memoize it somehow? Parsing these on every request is probably going to get expensive.